Source/Core/Common/FileUtil.cpp

```diff
@@ -651,9 +651,8 @@ void CopyDir(const std::string &source_path, const std::string &dest_path)
 // Returns the current directory
 std::string GetCurrentDir()
 {
-	char *dir;
-	// Get the current working directory (getcwd uses malloc)
-	if (!(dir = __getcwd(nullptr, 0)))
+	char* dir=(char*) malloc(PATH_MAX*sizeof(char));
```

Comment:
You never free `dir`. This should probably be on the stack instead of the heap, anyway...